qa/rgw/store/sfs/tests/create-s3tests-report.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
```

Comment:
lets drop this file too, we can keep it in the s3gw-tools repo if we still want, but I suspect this will never be used again.

Reply:
file dropped